### PR TITLE
Cleanup a lot of warnings that deal with LinkedList raw types.

### PR DESCRIPTION
**Pull request by BradWalker**
[repeat] /home/bwalker/src/netbeans/java/dbschema/src/org/netbeans/modules/dbschema/jdbcimpl/SchemaElementImpl.java:316: warning: [rawtypes] found raw type: LinkedList
   [repeat]         LinkedList result = new LinkedList<>();
   [repeat]         ^
   [repeat]   missing type arguments for generic class LinkedList<E>
   [repeat]   where E is a type-variable:

There are a few remaining ones that I did not touch. These are related to the methods being public and requiring changes or instances where
I would have to do major code work.

---
**^Add meaningful description above**

By opening a pull request you confirm that, unless explicitly stated otherwise, the changes -

 - are all your own work, and you have the right to contribute them.
 - are contributed solely under the terms and conditions of the Apache License 2.0 (see section 5 of the license for more information).

Please make sure (eg. `git log`) that all commits have a valid name and email address for you in the Author field.

If you're a first time contributor, see the Contributing guidelines for more information.

### CODE REVIEW

File: java/ant.debugger/src/org/netbeans/modules/ant/debugger/IOManager.java [hunk 1]
```diff
@@ -22,6 +22,7 @@
 import java.io.IOException;
 import java.util.Hashtable;
 import java.util.LinkedList;
+import java.util.List;
```
Comment (mklaehn) [inline java/ant.debugger/src/org/netbeans/modules/ant/debugger/IOManager.java:25]: I'm curious, why add the import if it is not used within the changeset?

**Review by mklaehn — APPROVED** [review]
Look good to me.
Noted one curiosity.

File: java/ant.debugger/src/org/netbeans/modules/ant/debugger/IOManager.java (hunk 1 above)
Comment (BradWalker) [reply java/ant.debugger/src/org/netbeans/modules/ant/debugger/IOManager.java:25]: I think that is an artifact of the work.. I'll fix it.. Thanks!

**Review by mklaehn — APPROVED** [review]
Great work

File: enterprise/j2ee.ddloaders/src/org/netbeans/modules/j2ee/ddloaders/multiview/CmpFieldsNode.java [hunk 2]
```diff
@@ -89,7 +89,7 @@ public SectionNode getNodeForElement(Object element) {
             }
         } else if (element instanceof CmpField[]) {
             final List list1 = Arrays.asList(cmpFields.getCmpFields());
-            final List list2 = new LinkedList(Arrays.asList((CmpField[]) element));
+            final List list2 = new LinkedList<>(Arrays.asList((CmpField[]) element));
```
Comment (mbien) [inline enterprise/j2ee.ddloaders/src/org/netbeans/modules/j2ee/ddloaders/multiview/CmpFieldsNode.java:92]: hi brad. I noticed a few declarations don't have a generic type on the left hand side. Was it not easily possible to assign one without breaking things?

Comment (BradWalker) [reply enterprise/j2ee.ddloaders/src/org/netbeans/modules/j2ee/ddloaders/multiview/CmpFieldsNode.java:92]: Hey @mbien , you are correct. I spent most of the time really trying to make the actual variables into a generic type and it was just not possible without also changing the public interfaces. Eventually, I'll get there, just not today. 8-)

**Pull request merged**